src/routes/about/+page.js

```diff
@@ -0,0 +1,229 @@
+const dataSrc = `https://api.github.com/repos/spyder-ide/spyder/contributors?per_page=100`;
+
+const currentContributors = [
+  {
+    id: 365293,
+    name: "Carlos Córdoba",
+    role: "Lead mantainer",
+    latino: true,
+  },
+  {
+    id: 16781833,
+    name: "Daniel Althviz",
+    role: "Co-mantainer",
+  },
+  {
+    id: 17051931,
+    name: "C.A.M. Gerlach",
+    role: "Documentation and technical writing",
+  },
+  {
+    id: 9618975,
+    name: "Ryan Clary",
+    role: "Installers",
+  },
+  {
+    id: 7941918,
+    name: "Jitse Niesen",
+    role: "External plugins",
+  },
+  {
+    id: 5204788,
+    name: "Hendrik D. Louzada",
+    role: "Remote development",
+    latino: true,
+  },
+  {
+    id: 42411448,
+    name: "Juan Sebastian Bautista",
+    role: "Junior Developer",
+    latino: true,
+  },
+  {
+    id: 5027583,
+    name: "Andrés Montoya",
+    role: "UI/UX Designer and Social Media",
+    latino: true,
+  },
+];
+
+const pastContributors = [
+  {
+    id: 1311787,
+    name: "Pierre Raybaut",
+    role: "Spyder Creator",
+  },
+  {
+    id: 1878982,
+    name: "Edgar Margffoy",
+    role: "LSP Support",
+  },
+  {
+    id: 50221806,
+    name: "Isabela Presedo-Floyd",
+    role: "UX/UI Redesign",
+    female: true,
+  },
+  {
+    id: 3627835,
+    name: "Gonzalo Peña-Castellanos",
+    role: "API Redesign",
+    latino: true,
+  },
+  {
+    id: 18587879,
+    name: "Juanita Gómez",
+    role: "Docs & Social Media",
+    latino: true,
+    female: true,
+  },
+  {
+    id: 10170372,
+    name: "Jean-Sébastien Gosselin",
+    role: "Plots pane",
+  },
+  {
+    id: 20992645,
+    name: "Stephannie Jimenez",
+    role: "Spyder-terminal maintainer",
+    latino: true,
+    female: true,
+  },
+  {
+    id: 2397974,
+    name: "Sylvain Corlay",
+    role: "New Icon Theme",
+  },
+  {
+    id: 2024217,
+    name: "Rafael Laverde",
+    role: "Editor improvements",
+    latino: true,
+  },
+  {
+    id: 10513354,
+    name: "Brian Olsen",
+    role: "Console improvements",
+  },
+];
+
+const textData = {
+  pageTitle: "Who We Are",
+  pageIntro: `Lorem ipsum dolor sit amet consectetur adipisicing elit. Nemo debitis
+        eligendi dolorem optio adipisci voluptatem inventore eius recusandae
+        reiciendis nisi aspernatur quas exercitationem nihil dolorum aliquam, ex
+        aliquid qui distinctio.`,
+  currentTitle: "Team Members",
+  pastTitle: "Former Team Members",
+  remainingTitle: "We are an open source project",
```

Comment:
```suggestion
  remainingTitle: "Contributor community",
```

Make more sense (especially in context of the other headings) and be more concise